src/renderer/components/icon/icon.tsx

```diff
@@ -43,6 +43,7 @@ export interface IconProps extends React.HTMLAttributes<any>, TooltipDecoratorPr
   focusable?: boolean;        // allow focus to the icon + show .active styles (default: "true", when icon is interactive)
   sticker?: boolean;
   disabled?: boolean;
+  title?: undefined;          // to suggest to TS users to use tooltip instead
```

Comment:
We shouldn't touch HTML native types. Replacing `title` to `tooltip` for icons should be enough.